spec/spec_helper.rb

```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```

Comment:
Clean up this file

Reply:
Cleaned up now.

Reply:
Not what I can see. Still need to get rid of the comments.

Reply:
Sorry, wrong file cleaned. Done now, promise.